tests/unit/test_metrics.py

```diff
@@ -71,6 +73,25 @@ def setUp(self):
         self.y_true = np.array([0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0])
         self.y_score = np.array([1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 1, 0, 0, 0])
 
+    @unittest.skip("Not ready to commit to requiring this in PR https://github.com/AstraZeneca/rexmex/pull/29")
```

Comment:
remove this if/when interested in committing to annotating all classification functions